rskj-core/src/main/java/org/ethereum/net/server/ChannelObserver.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * This file is part of RskJ
+ * Copyright (C) 2023 RSK Labs Ltd.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.ethereum.net.server;
+
+public class ChannelObserver {
+
+    private ChannelObserver() {
+    }
+
+    public interface Observable {
```

Comment:
Is it necessary to declare the interface here?

Reply:
Just wanted to keep both interfaces in the same file for simplicity. Since they are really small, I thought creating an individual file for each was not ideal.